rfq/manager.go

```diff
@@ -1212,6 +1215,61 @@ func (m *Manager) RfqChannel(ctx context.Context,
 	return balancesMap, nil
 }
 
+// InboundPolicyFetcher is a helper that fetches the inbound policy of a channel
+// based on its chanID.
+type InboundPolicyFetcher func(ctx context.Context, chanID uint64,
+	remotePubStr string) (*lnrpc.RoutingPolicy, error)
+
+// RfqToHopHint creates the hop hint representation which encapsulates certain
+// quote information along with some other data required by the payment to
+// succeed.
+func (m *Manager) RfqToHopHint(ctx context.Context,
+	policyFetcher InboundPolicyFetcher, channelID uint64,
+	peerPubKey route.Vertex, quote *rfqrpc.PeerAcceptedBuyQuote,
+	hold bool) (*lnrpc.HopHint, []zpay32.HopHint, error) {
```

Comment:
We do we return a single RPC hop hint, but a slice of normal hop hints? 

I think the `godoc` comment could use some expansion. Going a bit futher, we may want to consider making a new wrapper type here depending on the end use case (does the client sometimes need to map between the two versions?).